src/NzbDrone.Core/DecisionEngine/QualityUpgradableSpecification.cs

```diff
@@ -6,8 +7,8 @@ namespace NzbDrone.Core.DecisionEngine
 {
     public interface IQualityUpgradableSpecification
```

Comment:
Completly agree. I think IUpgradableSpecification is more inline with the work that class is doing atm.